### PR TITLE
Clean up & Organize Router

### PR DESCRIPTION
**Pull request by ryanw-ny**
Everything is nice and tidy, and in separate packages, to avoid confusion while working with it.

### CODE REVIEW

**Comment by ryanw-ny** [comment]
@thisandagain This should be good to go now!

File: app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java [hunk 1]
```diff
@@ -2,29 +2,27 @@
...
-        return singleton;
-    }
 
     @Override
     public void onCreate() {
         super.onCreate();
         singleton = this;
 
-        Log.v("[Webmaker]", "Application created.");
+        Log.v("[WebMaker]", "Application created.");
```
Comment (thisandagain) [inline app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java]: "Webmaker" the brand is never PascalCase.

File: app/src/main/java/mozilla/org/webmaker/router/Router.java [hunk 2]
```diff
@@ -160,24 +79,21 @@ public void map(String format, RouterCallback callback) {
     /**
      * Map a URL to open an {@link Activity}
      * @param format The URL being mapped; for example, "users/:id" or "groups/:id/topics/:topic_id"
-     * @param klass The {@link Activity} class to be opened with the URL
+     * @param clazz The {@link Activity} class to be opened with the URL
      */
-    public void map(String format, Class<? extends Activity> klass) {
-        this.map(format, klass, null);
+    public void map(String format, Class<? extends Activity> clazz) {
+        this.map(format, clazz, null);
```
Comment (thisandagain) [inline app/src/main/java/mozilla/org/webmaker/router/Router.java]: LOL. :+1:

File: app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java (hunk 1 above)
Comment (ryanw-ny) [inline app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java]: @thisandagain Not sure how I missed that.. will fix!

**Comment by thisandagain** [comment]
Cool. So I think the only question that this raises is documenting how devs can trigger `router.open` outside of the `WebmakerApplication` class. We'll need this in order to build up the interface to JS this week. Mind adding some minimal docs to the top of the Router?

**Pull request closed without merging**